tutorial_R/raincloud_tutorial_r.Rmd

```diff
@@ -45,9 +48,10 @@ p1 <- ggplot(summary_simdat, aes(x = group, y = score_mean, fill = group))+
   ylim(0, 80)+
   ylab('Score')+xlab('Group')+theme_cowplot()+
   ggtitle("Figure 1: Barplot +/- SEM")
-  ggsave('../figs/tutorial_R/1Barplot.png', width = w, height = h)
 
-  p1
+ggsave('../figs/tutorial_R/1Barplot.png', width = w, height = h)
+
+p1
```

Comment:
All these `ggsave` differences are just linting because they don't need to be indented, right? Does it change the behaviour of the code at all?

Reply:
Ah - I've had a click through binder and they it seems that removing *this* indent doesn't matter but the next few cells don't show the image inline.

I can fix the behaviour by re-indenting the `ggsave` and `p1` lines (and all subsequent pairs in the following cells)....I'm not sure what it's doing but here's an error output....

![image](https://user-images.githubusercontent.com/3626306/44710715-ad151a80-aaa4-11e8-92e7-fde66a6171c9.png)

Reply:
URGH - just kidding - it's super unclear to me when I get this error and when I don't..... @ha0ye - if you have any ideas please let me know, but otherwise I'll have a closer look when I have time. The `knit` button works fine, it's probably me not knowing what `Run` does 😩

Reply:
I think this is a timing issue with running the Rmarkdown file - the last line tries to output the plot inline, but it hasn't finished rendering the plot to a PNG for display. I can play around to find a solution, but usually just waiting for a bit and then running again works.

You could also cache the output of the code chunk so that it's not an issue.